features/steps/tabstops.py

```diff
@@ -78,3 +79,43 @@ def then_the_tab_stops_are_sequenced_in_position_order(context):
     tab_stops = context.tab_stops
     for idx in range(len(tab_stops) - 1):
         assert tab_stops[idx].position < tab_stops[idx+1].position
+
+
+@then('the tab stop at index {index} has position {position}')
+def then_the_tab_stop_at_index_index_has_position_position(context, index,
+                                                           position):
+    tab_stops = context.tab_stops
+    assert tab_stops[int(index)].position == int(position)
+
+
+@then('the tab stop at index {index} has alignment {alignment}')
+def then_the_tab_stop_at_index_index_has_alignment_alignment(context, index,
+                                                             alignment):
+    tab_stops = context.tab_stops
+    new_alignment = {
+        'WD_TAB_ALIGNMENT.LEFT':    WD_TAB_ALIGNMENT.LEFT,
+        'WD_TAB_ALIGNMENT.CENTER':  WD_TAB_ALIGNMENT.CENTER,
+        'WD_TAB_ALIGNMENT.RIGHT':   WD_TAB_ALIGNMENT.RIGHT,
+        'WD_TAB_ALIGNMENT.DECIMAL': WD_TAB_ALIGNMENT.DECIMAL,
+        'WD_TAB_ALIGNMENT.BAR':     WD_TAB_ALIGNMENT.BAR,
+        'WD_TAB_ALIGNMENT.LIST':    WD_TAB_ALIGNMENT.LIST,
+        'WD_TAB_ALIGNMENT.CLEAR':   WD_TAB_ALIGNMENT.CLEAR,
+        'WD_TAB_ALIGNMENT.END':     WD_TAB_ALIGNMENT.END,
+        'WD_TAB_ALIGNMENT.NUM':     WD_TAB_ALIGNMENT.NUM,
+        'WD_TAB_ALIGNMENT.START':   WD_TAB_ALIGNMENT.START,
+    }[alignment]
+    assert tab_stops[int(index)].alignment == new_alignment
+
+
+@then('the tab stop at index {index} has leader {leader}')
+def then_the_tab_stop_at_index_index_has_leader_leader(context, index, leader):
+    tab_stops = context.tab_stops
+    new_leader = {
+        'WD_TAB_LEADER.SPACES':     None,  # WD_TAB_LEADER.SPACES,
+        'WD_TAB_LEADER.DOTS':       WD_TAB_LEADER.DOTS,
+        'WD_TAB_LEADER.DASHES':     WD_TAB_LEADER.DASHES,
+        'WD_TAB_LEADER.LINES':      WD_TAB_LEADER.LINES,
+        'WD_TAB_LEADER.HEAVY':      WD_TAB_LEADER.HEAVY,
+        'WD_TAB_LEADER.MIDDLE_DOT': WD_TAB_LEADER.MIDDLE_DOT,
+    }[leader]
```

Comment:
In these situations where an enumeration is needed, I've adopted a more compact approach. The existing acceptance tests are a mixed bag on this count because I haven't gone back to update them all, but new work can use this more compact method. Here's the gist by way of example:

```cucumber
...
Then tab_stop.alignment is <value>
...

Examples: tab stop positions
  | position |
  | LEFT     |
...
```

```python
def then_tab_stop_alignment_is_value(context, value):
    expected_value = None if value == 'None' else getattr(WD_TAB_LEADER, value)
```

So we just use the string name of the enumeration member, and resolve that to the particular reference in the step code.